whitelabelling_bar/package.json

```diff
@@ -0,0 +1,130 @@
+    },
+    "peerDependencies": {
+        "@gooddata/sdk-model": "9.3.0",
+        "@gooddata/sdk-backend-bear": "9.3.0",
+        "@gooddata/sdk-backend-spi": "9.3.0",
+        "@gooddata/sdk-backend-tiger": "9.3.0",
+        "@gooddata/sdk-ui": "9.3.0",
+        "@gooddata/sdk-ui-charts": "9.3.0",
+        "@gooddata/sdk-ui-ext": "9.3.0",
+        "@gooddata/sdk-ui-filters": "9.3.0",
+        "@gooddata/sdk-ui-geo": "9.3.0",
+        "@gooddata/sdk-ui-kit": "9.3.0",
+        "@gooddata/sdk-ui-pivot": "9.3.0",
+        "react": "^16.10.0 || ^17.0.0 || ^18.0.0",
+        "react-dom": "^16.10.0 || ^17.0.0 || ^18.0.0"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.7.2",
+        "@babel/preset-env": "^7.7.2",
+        "@babel/preset-react": "^7.7.2",
+        "@gooddata/catalog-export": "9.3.0",
+        "@gooddata/frontend-npm-scripts": "^1.3.0",
+        "@gooddata/plugin-toolkit": "9.3.0",
+        "@gooddata/sdk-backend-bear": "9.3.0",
+        "@gooddata/sdk-backend-spi": "9.3.0",
+        "@gooddata/sdk-backend-tiger": "9.3.0",
+        "@gooddata/sdk-model": "9.3.0",
+        "@gooddata/sdk-ui": "9.3.0",
+        "@gooddata/sdk-ui-charts": "9.3.0",
+        "@gooddata/sdk-ui-ext": "9.3.0",
+        "@gooddata/sdk-ui-filters": "9.3.0",
+        "@gooddata/sdk-ui-geo": "9.3.0",
+        "@gooddata/sdk-ui-kit": "9.3.0",
+        "@gooddata/sdk-ui-loaders": "9.3.0",
+        "@gooddata/sdk-ui-pivot": "9.3.0",
+        "@types/json-stable-stringify": "^1.0.32",
+        "@types/lodash": "^4.14.158",
+        "@types/node": "^16.18.23",
+        "@types/raf": "^3.4.0",
+        "@types/react": "^18.2.0",
+        "@types/react-dom": "^18.2.1",
+        "@typescript-eslint/eslint-plugin": "^5.57.1",
+        "@typescript-eslint/parser": "^5.57.1",
+        "babel-loader": "^8.0.5",
+        "case-sensitive-paths-webpack-plugin": "^2.4.0",
+        "concurrently": "^6.0.2",
+        "css-loader": "^6.7.1",
+        "dotenv": "16.0.3",
+        "dotenv-webpack": "^7.0.2",
+        "eslint": "^8.25.0",
+        "eslint-import-resolver-typescript": "^3.6.0",
+        "eslint-plugin-import": "^2.22.0",
+        "eslint-plugin-import-esm": "^1.2.1",
+        "eslint-plugin-prettier": "^4.0.0",
+        "eslint-plugin-react": "^7.31.10",
+        "eslint-plugin-react-hooks": "^4.5.0",
+        "html-webpack-plugin": "^5.3.1",
+        "lodash": "^4.17.19",
+        "prettier": "~2.5.0",
+        "raf": "^3.4.1",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-intl": "^6.4.1",
+        "rimraf": "^3.0.2",
+        "source-map-loader": "^4.0.0",
+        "style-loader": "^3.3.1",
+        "ts-loader": "^8.3.0",
+        "typescript": "^5.2.2",
+        "vitest": "0.31.4",
+        "webpack": "^5.58.0",
+        "webpack-bundle-analyzer": "^4.5.0",
+        "webpack-cli": "^4.9.2",
+        "webpack-dev-server": "^4.9.1"
+    },
+    "resolutions": {
+        "reselect": "4.1.5"
+    },
+    "lint-staged": {
```

Comment:
@dremileta as discussed, since lint-staged would require husky setup etc., let's not bring this here at this point.